modules/api/php/endpoints/candidate/candidate.class.inc

```diff
@@ -10,11 +10,11 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris
  */
-namespace LORIS\Api\Endpoints\Candidate;
+namespace LORIS\api\Endpoints\Candidate;
```

Comment:
Hey @xlecours wouldn't it make more sense if api was in all caps?
`namespace LORIS\API\Endpoints\Candidate`

Reply:
all our modules name are lower case. I agree that it's an acronyme like LORIS and should probably be all caps but I will stick with the module name convention.